aten/src/ATen/native/Convolution.cpp

```diff
@@ -119,6 +121,21 @@ auto ConvParams::view1d_as_2d() -> void {
   }
 }
 
+bool ConvParams::use_caffe2_depthwise(
+    const at::Tensor& input, const at::Tensor& weight) const
+{
+  // Currently only 3x3 depthwise convolutions on tensors of float are supported.
+  return !transposed &&
+         !is_dilated() &&
+         (input.scalar_type() == at::kFloat) &&
+         (input.ndimension() == 4) &&
+         (input.size(1) == groups) &&
+         (weight.scalar_type() == at::kFloat) &&
+         (weight.size(0) % input.size(1) == 0) &&
+         (weight.sizes()[2] == 3) &&
```

Comment:
Why is there a mix of `size(n)` and `sizes()[n]`?  Seems like the latter is what you want?

Reply:
Thanks for pointing that out.  I am still not very familiar with the codebase so keep making writing unorthodox code.  Please keep pointing these out.